intro.tex

```diff
@@ -1,41 +1,54 @@
 \section{Introduction}
 
-This  note describes the plan for ensuring the Rubin community will have sufficient data access, data products, and data analysis tools to produce early science when the full Legacy Survey of Space and Time (LSST)  begins. The start date for full survey operations is planned for April 1, 2024 with considerable uncertainty. As of November 2021, the construction project schedule shows completetion between December 2023 and July 2024 (i.e. planned finish and finish with contingency). It is unlikely the project will finish in December 2023 and highly likely it will finish by July 2024. 
+This  note describes the plan for ensuring the Rubin community will have the data products and services necessary to produce early science when the full Legacy Survey of Space and Time (LSST)  begins. 
+The current planned start date for full survey operations is 1 April 2024, with considerable uncertainty. 
+As of November 2021, the construction project schedule shows completion between December 2023 and July 2024 (i.e. planned finish and finish with contingency). 
+It is unlikely the project will finish in December 2023 and highly likely it will finish by July 2024. 
 
 \subsection{Definition of Early Science}  \label{ssec:defn}
 
-Given that a first release of catalog data in operations requires significant progress in year 1, incuding generation of templates for alert production, Early Science (ES) is defined as any science enabled by Rubin for its community that happens before Data Release 1 (DR1) at the end of the first year of full survey operations. 
-Community expectations for early science are high due to the transformative nature of the Rubin data.  
-Additionally, for many science goals, time-sensitive follow-up observations are needed to take full advantage of the Rubin data.
+Early Science (ES) is defined as any science enabled by Rubin for its community through and including the first data release, \drone.
+DR1 will be based on the first 6 months of data and is scheduled for release 12 months after the start of operations, \citep{RDO-011}.
```

Comment:
maybe
... after the start of full survey operations (i.e. data taking on the LSST).